lib/config.js

```diff
@@ -1423,6 +1423,21 @@ util.stripYamlComments = function(fileStr) {
 }
 
 /**
+ * Strip all Javascript type comments from JSON strings.
+ *
+ * This method will call util.stripComments but has a simplified regex that
+ * will work with JSON strings.
+ *
+ * @protected
+ * @method stripJSONComments
+ * @param fileString {string} The string to strip comments from
+ * @return {string} The string with comments stripped.
+ */
+util.stripJSONComments = function(fileStr) {
+  return util.stripComments(fileStr, /("[^"]*")/g);
```

Comment:
I'm not a regex guru.  Could you add a comment here describing this regex?  Something like
```
// Match everything starting with a quote up to and including a matching quote.
```
It appears that this would miss a lot - such as /* ... */ comments, but I can't tell without a good description of the intention of this regex.